task34.py

```diff
@@ -0,0 +1,9 @@
+# Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм
+# 'Парам пам-пам' - есть ритм, 'Пам парам' - нет ритма
+
+poem_list = input('стих: ').lower().split()
+vows_count_list = list(map(lambda str: sum(1 for i in str if i in 'аяуюоеёэиы'), poem_list))
```

Comment:
Можно сделать так
```suggestion
vows_count_list = list(map(lambda str: sum(i in 'аяуюоеёэиы' for i in str), poem_list))
```
Кстати, использовать переменную `str` – не самая лучшая идея.